### PR TITLE
Add support for lnn, amm, and aga

### PR DESCRIPTION
**Pull request by cdmistman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,16 +22,49 @@
...
             }
         ],
         "grammars": [
             {
                 "language": "alan",
                 "scopeName": "source.ln",
                 "path": "./syntaxes/alan.tmLanguage.json"
+            },
+            {
+                "language": "alan microcode",
```
Comment (depombo) [inline package.json]: nit: `amm` stands for alan minus minus

**Review by depombo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (depombo) [reply package.json]: as a pun on `C++`

Comment (cdmistman) [reply package.json]: ohhhhh oops

**Review by dfellis — APPROVED** [review]
Love to see parity with the vim syntax highlighting :smile:

**Review by aguillenv — APPROVED** [review]

**Pull request merged**